crates/jstz_cli/src/logs/mod.rs

```diff
@@ -9,36 +10,17 @@ mod trace;
 pub enum Command {
     /// View logs
     Trace {
-        /// View console.log messages
-        #[arg(long)]
-        log: bool,
-        /// View console.info messages
-        #[arg(long)]
-        info: bool,
-        /// View console.warn messages
-        #[arg(long)]
-        warn: bool,
-        /// View console.error messages
-        #[arg(long)]
-        error: bool,
-        /// View contract creations
-        #[arg(long)]
-        contract: bool,
-        /// Add custom search strings
-        #[arg(long, value_name = "custom_strings")]
-        custom: Vec<String>,
+        #[arg(value_name = "SMART_FUNCTIOIN")]
+        address: String,
     },
 }
 
-pub fn exec(command: Command, cfg: &mut Config) -> Result<()> {
+pub async fn exec(command: Command, cfg: &mut Config) -> Result<()> {
     match command {
-        Command::Trace {
-            log,
-            info,
-            warn,
-            error,
-            contract,
-            custom,
-        } => trace::exec(log, info, warn, error, contract, custom, cfg),
+        Command::Trace { address } => {
+            let address = PublicKeyHash::from_base58(&address)
```

Comment:
Can you please make it so that it works with aliases, so that the address value can either be an alias or an address? 
Using AccountConfig (cfg.accounts.get), you can check whether an account exists for a particular address. If it does, use the account, otherwise try to convert it into an address.